doc/modules/model_evaluation.rst

```diff
@@ -976,9 +977,16 @@ In multi-label classification, the :func:`roc_auc_score` function is
 extended by averaging over the labels as :ref:`above <average>`.
 
 Compared to metrics such as the subset accuracy, the Hamming loss, or the
-F1 score, ROC doesn't require optimizing a threshold for each label. The
-:func:`roc_auc_score` function can also be used in multi-class classification,
-if the predicted outputs have been binarized.
+F1 score, ROC doesn't require optimizing a threshold for each label.
+
+The :func:`roc_auc_score` function can also be used in multi-class
+classification. Two averaging strategies are currently supported: the
+Hand & Till (2001) one-vs-one algorithm computes the average of the pairwise
+ROC AUC scores, and the Provost & Domingos (2001) one-vs-rest algorithm
+computes the average of the ROC AUC scores for each class against
+all other classes. In both cases, the predicted class labels are provided in
+an array with values from 0 to `n_classes`, and the scores correspond to the
+probability estimates that a sample belongs to a particular class.
```

Comment:
Not really required but you could try to make an example that compares the two approaches.